src/python/datastore/data_types.py

```diff
@@ -1108,6 +1108,16 @@ def _pre_put_hook(self):
                        fuzz_target_job_key(self.fuzz_target_name, self.job))
 
 
+class FuzzStrategyProbability(Model):
+  """Mapping between fuzz strategies and probabilities with which they
+  should be selected
+
+  Calculated using a softmax multi-armed bandit probability model."""
+  strategy_name = ndb.StringProperty()
+  strategy_count = ndb.IntegerProperty()
```

Comment:
I should have asked this earlier, but is there any reason we'd need the count later? I'd expect this to be unnecessary for the bots.